<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="xuxiaochan.test.airwallex.BankInfoEndpointTest" tests="73" time="31.766" ignored="0" failures="19" timestamp="24 七月 2019 10:47:37 GMT" skipped="0" hostname="xuxiaochande-MacBook-Pro.local" errors="0">
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.449" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.407" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.602" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of account_number should be between 6 and 9 when bank_country_code is &#039;AU&#039;&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of account_number should be between 6 and 9 when bank_country_code is 'AU'"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.615" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.526" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.377" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of account_number should be between 8 and 20 when bank_country_code is &#039;CN&#039;&quot;}] but found [{&quot;success&quot;:&quot;Bank details saved&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of account_number should be between 8 and 20 when bank_country_code is 'CN'"}] but found [{"success":"Bank details saved"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.512" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.404" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.401" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;&#039;aba&#039; is required when bank country code is US&quot;}] but found [{&quot;success&quot;:&quot;Bank details saved&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"'aba' is required when bank country code is US"}] but found [{"success":"Bank details saved"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.505" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.614" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of account_number should be between 8 and 20 when bank_country_code is &#039;CN&#039;&quot;}] but found [{&quot;success&quot;:&quot;Bank details saved&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of account_number should be between 8 and 20 when bank_country_code is 'CN'"}] but found [{"success":"Bank details saved"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.414" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.511" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.412" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.511" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of account_number should be between 6 and 9 when bank_country_code is &#039;AU&#039;&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of account_number should be between 6 and 9 when bank_country_code is 'AU'"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.449" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;The swift code is not valid for the given bank country code: CN&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"The swift code is not valid for the given bank country code: CN"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.510" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.405" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.360" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.370" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;The swift code is not valid for the given bank country code: CN&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"The swift code is not valid for the given bank country code: CN"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.418" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.524" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.338" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of &#039;swift_code&#039; should be either 8 or 11&quot;}] but found [{&quot;success&quot;:&quot;Bank details saved&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of 'swift_code' should be either 8 or 11"}] but found [{"success":"Bank details saved"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.613" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.419" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.371" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of account_number should be between 8 and 20 when bank_country_code is &#039;CN&#039;&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of account_number should be between 8 and 20 when bank_country_code is 'CN'"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.368" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.411" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.407" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.347" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;success&quot;:&quot;Bank details saved&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"success":"Bank details saved"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.444" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.411" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.896" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.515" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of &#039;swift_code&#039; should be either 8 or 11&quot;}] but found [{&quot;success&quot;:&quot;Bank details saved&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of 'swift_code' should be either 8 or 11"}] but found [{"success":"Bank details saved"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.GeneratedMethodAccessor1.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.393" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.401" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.410" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
  <testcase name="bankInfoCollectEndpointRequest" time="0.409" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest">
    <failure type="java.lang.AssertionError" message="expected [{&quot;error&quot;:&quot;Length of account_number should be between 1 and 17 when bank_country_code is &#039;US&#039;&quot;}] but found [{&quot;error&quot;:&quot;Length of account_number should be between 7 and 11 when bank_country_code is &#039;US&#039;&quot;}]">
      <![CDATA[java.lang.AssertionError: expected [{"error":"Length of account_number should be between 1 and 17 when bank_country_code is 'US'"}] but found [{"error":"Length of account_number should be between 7 and 11 when bank_country_code is 'US'"}]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at xuxiaochan.test.airwallex.BankInfoEndpointTest.bankInfoCollectEndpointRequest(BankInfoEndpointTest.java:140)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.TestNG.privateMain(TestNG.java:1434)
	at org.testng.TestNG.main(TestNG.java:1403)
]]>
    </failure>
  </testcase> <!-- bankInfoCollectEndpointRequest -->
  <testcase name="bankInfoCollectEndpointRequest" time="0.408" classname="xuxiaochan.test.airwallex.BankInfoEndpointTest"/>
</testsuite> <!-- xuxiaochan.test.airwallex.BankInfoEndpointTest -->
